atcoder.js

```diff
@@ -0,0 +1,52 @@
+window.browser = (function () {
+    return window.msBrowser ||
+      window.browser ||
+      window.chrome;
+  })();
+
+var url = window.location.href;
+var submissions = [].slice.call(document.getElementsByClassName("waiting-judge")).map(it => it.getAttribute("data-id"));
+
+function fetch_atcoder_result(base_url, ids) {
+    var xhr = new XMLHttpRequest();
+    var url_param = [].slice.call(ids).map(it => "sids[]=" + ids).join("&")
```

Comment:
Shouldn't you use it here?
```suggestion
    var url_param = [].slice.call(ids).map(it => "sids[]=" + it).join("&")
```